resources/js/components/FormConditions.vue

```diff
@@ -11,10 +11,15 @@ export default {
         callback: {
             type: Function,
         },
+        ajaxResponse: {
+            type: Boolean,
+            default: false
+        }
     },
     data() {
         return {
             formData: this.initialData,
+            ajaxResponse: this.ajaxResponse,
```

Comment:
This is not needed, if it's a prop you can just use it as this.ajaxResponse without declaring it in the data